src/main/scala/com/github/unchama/buildassist/MenuInventoryData.scala

```diff
@@ -620,45 +387,60 @@ object MenuInventoryData {
     itemstack.setItemMeta(skullmeta)
     AsyncInventorySetter.setItemAsync(inventory, 45, itemstack)
 
+    // 4ページ目を開く
+    itemstack = new ItemStack(Material.SKULL_ITEM, 1, 3)
```

Comment:
```suggestion
	// 3 = プレイヤーの頭
    itemstack = new ItemStack(Material.SKULL_ITEM, 1, 3)
```
(これはCircleCIにビルドさせるための変更)